### PR TITLE
Android tabbar replace elevation with border

### LINKED ISSUE #10604: Disable elevation on Android Tab Bar, replace it with a 1pt divider
# Problem

We can't seem to achieve the desired outcome using elevation values on Android's bottom tab bar, this might be acceptable in light mode but makes the shadow completely unnoticeable in dark mode.

## Implementation

1. Disable elevation on Tab Bar on Android
2. Add a 1px divider, Light Grey (and dark theme equivalent), aligned to the top of the Top Bar, width equals Top Bar width.

Screenshot 2020-05-07 at 15 49 29

## Acceptance Criteria

Elevation is replaced by a divider

## Notes

Elevation can suck a donkey ball

### PR DESCRIPTION
**Pull request by Ferossgp**
Fixes #10604

### CODE REVIEW

File: src/status_im/ui/components/tabbar/styles.cljs [hunk 1]
```diff
@@ -78,24 +78,27 @@
...
-   :position         :absolute
-   :left             0
-   :right            0
-   :height           tabs-height
-   :bottom           inset
-   :transform        [{:translateY
-                       (animation/interpolate visible?
-                                              {:inputRange  [0 1]
-                                               :outputRange [0 tabs-diff]})}]})
+  (merge
```
Comment (flexsurfer) [inline src/status_im/ui/components/tabbar/styles.cljs:78]: you can use `styles/defn` macro

**Review by flexsurfer — APPROVED** [review]

**Comment by Serhy** [comment]
Looks good on Androids

**Comment by flexsurfer** [comment]
@Ferossgp can we merge this one ?

**Comment by Ferossgp** [comment]
@flexsurfer I'm experimenting with other approaches to change shadow in android, let's skip this PR for 1.5

**Comment by oskarth** [comment]
Small request - could you please rebase to latest master so this commit https://github.com/status-im/status-react/commit/b5fda1234e5447cbc626059dfc78fcfbabc9e926 gets included? This means we aren't running E2E tests on eth.prod cluster anymore, and since this happens on every push, the impact is quite big. This ensures we have more accurate metrics going forward from Jul 1 onward, which would be awesome. See https://discuss.status.im/t/user-growth-and-retention/1782 for more

**Comment by flexsurfer** [comment]
@Ferossgp any updates on this ?

**Comment by Serhy** [comment]
Checked with Android 6, 8, 9, 10 - no gradient anymore with dark and light mode. And it shows on iOS. 
Thanks, @Ferossgp

**Pull request merged**